connect/api_test.go

```diff
@@ -41,7 +45,11 @@ func TestGetActivationsRequest(t *testing.T) {
 		url        = "/connect/systems/activations"
 		gotRequest *http.Request
 	)
-
+	CFG.FsRoot = t.TempDir()
+	if err := writeSystemCredentials(user, password); err != nil {
+		t.Errorf("Unexpected error: %s", err)
+		return
```

Comment:
Will change these to use t.Fatalf() and then no need for return

Reply:
Actually a helper function might be better. Like:
createTestCredentials(user, pass string, t *testing.T)